release-please-config.json

```diff
@@ -0,0 +1,23 @@
+{
+    "bootstrap-sha": "5ec5c3163f499caef4cfe2a62cdedb27ef3ce170",
+    "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
+    "pull-request-header": ":rocket: A new MSGraph Core TypeScript SDK release is coming up!",
+    "include-component-in-tag": false,
+    "versioning": "prerelease",
+    "packages": {
+        ".": {
+            "package-name": "msgraph-ts-core-sdk",

```

Comment:
how did you come up with this value?

Reply:
the npm package name is `@microsoft/msgraph-sdk-core` I'd suggest we align with that or the repo name to avoid any potential confusion. Does this setting feed into anything ? (tag names, changelog, etc...)

Reply:
According to https://github.com/googleapis/release-please/blob/3202daa81d0a5ef8f8959ec48aa5f79159fde997/docs/manifest-releaser.md?plain=1#L258, we don't need to set the value here for 'node' releases. It will be inferred from the `package.json` file [here](https://github.com/googleapis/release-please/blob/3202daa81d0a5ef8f8959ec48aa5f79159fde997/src/strategies/node.ts#L93).

However, as @baywet states, it should ideally match name of the package we release over in npm,